lib/cc/engine/lockfile_specs.rb

```diff
@@ -1,15 +1,14 @@
-require "bundler"
-
 class LockfileSpecs
   def initialize(lockfile)
-    @contents = File.read(lockfile) if File.exist?(lockfile)
+    @lockfile = lockfile
   end
 
   def include?(name)
-    return false if @contents.nil?
-    @lockfile ||= Bundler::LockfileParser.new(@contents)
-    @lockfile.specs.any? { |spec| spec.name == name }
-  rescue Bundler::LockfileError
-    false
+    if File.exist?(@lockfile)
+      @contents ||= File.read(@lockfile)
+      @contents.lines.any? { |line| line =~ /#{name}/ }
```

Comment:
I think if you use `File.open` you can avoid loading the whole file into memory.

```rb
File.open(@lockfile) { |f| f.lines.any? { ... } }
```

If that matters